src/Breadcrumbs.tsx

```diff
@@ -79,9 +79,16 @@ const BreadcrumbItem = styled.a.attrs<StyledBreadcrumbItemProps>(props => ({
   ${sx};
 `
 
-Breadcrumb.displayName = 'Breadcrumb'
+Breadcrumbs.displayName = 'Breadcrumbs'
 
-BreadcrumbItem.displayName = 'Breadcrumb.Item'
+BreadcrumbsItem.displayName = 'Breadcrumbs.Item'
 
-export type BreadcrumbItemProps = ComponentProps<typeof BreadcrumbItem>
-export default Object.assign(Breadcrumb, {Item: BreadcrumbItem})
+export type BreadcrumbsItemProps = ComponentProps<typeof BreadcrumbsItem>
+export default Object.assign(Breadcrumbs, {Item: BreadcrumbsItem})
+
+/**
+ * @deprecated Use the Breadcrumbs component instead (i.e. <Breadcrumb> → <Breadcrumbs>)
+ */
+export const Breadcrumb = Object.assign(Breadcrumbs, {Item: BreadcrumbsItem})
+export type BreadcrumbProps = ComponentProps<typeof BreadcrumbsBase>
```

Comment:
Can we add deprecation warnings for these types as well?

Reply:
Sure thing! I followed the same approach as when Position subcomponents where deprecated (component had depreciation warning but not the props types). In any case makes sense to see the message in the IDE if you're importing the outdated types.